Doc/library/ast.rst

```diff
@@ -146,6 +146,98 @@ Node classes
     Snakes <https://greentreesnakes.readthedocs.io/en/latest/>`__ project and
     all its contributors.
 
+
+.. _ast-root-nodes:
+
+Root nodes
+^^^^^^^^^^
+
+.. class:: Module(body, type_ignores)
+
+   A Python module.
```

Comment:
```suggestion
   A Python module, as with :ref:`file input <file-input>`.
   Node type generated by :func:`ast.parse` in the default ``"exec"`` *mode*.
```

May as well be explicit here, too, and consistent with the others, by mentioning the parsing mode.

Also, explicitly state and cross-reference the language reference for more details, like the others.